package.json

```diff
@@ -30,9 +30,10 @@
     "test:unit": "lerna run test:unit",
     "test:lint": "lerna exec -- ../../node_modules/.bin/prettier --write --print-width 100 --tab-width 2 --no-semi --single-quote --trailing-comma es5 --parser typescript src/**/*.ts",
     "build": "node tools/build.js",
+    "changelog": "github_changelog_generator --no-compare-link --bug-labels 'Type:\u0020Bug' --enhancement-labels 'Type:\u0020Feature,Type:\u0020Chore' --include-labels 'Type:\u0020Feature,Type:\u0020Chore,Type:\u0020Bug,Type:\u0020Breaking,Type:\u0020Regression' --user motorcyclets --project motorcycle",
```

Comment:
why these `\u0020` codes?

Reply:
They are spaces, it doesn't seem to work with _actual_ spaces in their place. It errors trying to read them.

Reply:
I thought so.